### PR TITLE
Feat/post media

### LINKED ISSUE #52: Component: Post Media
Style the media components that appear in posts. 

- [ ] Images
- [ ] Podcasts
- [ ] Videos
- [ ] Interactives - This is a lazy block, so you will need to add the correct classes and adjust the HTML in the WP admin under Lazy Blocks -> Blocks -> Post Interactive.
- [ ] Blockquotes - This is a lazy block, so you will need to add the correct classes and adjust the HTML in the WP admin under Lazy Blocks -> Blocks -> Blockquote.

### PR DESCRIPTION
**Pull request by HP4k1h5**
Creates scss class and lazyblock code for media components in posts.

Closes #52

### CODE REVIEW

**Review by urchykli — CHANGES_REQUESTED** [review]
Nice first pass on this, Bob!  In addition to the comment below about alignments, I left a few inline comments.

When someone adds a blockquote to a post, they should be choosing between left and right aligned.  Since we can't force them to pick one, the default should be right aligned and if the others are picked it should align accordingly.  Also, when the alignments are added, they are applied at a smaller breakpoint than they should be.  This is because other elements that have these alignements (images, etc.) float at smaller breakpoints.  That being said, you'll need to adjust the `--margin-text-side` and --margin-gutter-side` variables in this file to work with the alignments [here](https://github.com/CSIS-iLab/nuclearnetwork_wp/blob/cd1b283c23447a2f4d2f47de3a3a4dfa2553b6cc/wp-content/themes/csisnuclearnetwork/assets/_scss/blocks/post.scss#L37).

File: wp-content/themes/csisnuclearnetwork/assets/_scss/blocks/post/_blockquote.scss [hunk 1]
```diff
@@ -2,36 +2,44 @@
...
+      color: $color-black-190;
+      font-weight: 400;
     }
 
     &__author {
-      color: $color-white-165;
+      margin: rem(12) 0 rem(32) 0;
       @extend %text-style-paragraph-medium-short-caps;
+      color: $color-black-165;
       font-style: normal;
```
[the remark this answers is not in the document]
Comment (HP4k1h5) [reply wp-content/themes/csisnuclearnetwork/assets/_scss/blocks/post/_blockquote.scss:27]: author name appears in italics without `font-style: normal`, but i can't see why looking up the inheritance chain.

[the remark this answers is not in the document]
Comment (urchykli) [reply wp-content/themes/csisnuclearnetwork/assets/_scss/blocks/post/_blockquote.scss:31]: Ah good catch! It's the default style for `cite`

**Review by urchykli — CHANGES_REQUESTED** [review]
We're getting there!

When I said, you'll need to adjust the `--margin-text-side` and `--margin-gutter-side` variables, I meant for the small and medium breakpoints. Since it should be centered on these screen sizes, both of the margin variables should be set to `auto` and the float should be removed.  

Also, the `max-width` doesn't apply until the medium breakpoint.

Example of the margins:
```
@include breakpoint('small') {
      --margin-text-side: auto;
      --margin-gutter-side: auto;
      float: none !important;
      max-width: 100%;
    }

@include breakpoint('medium') {
      --margin-text-side: auto;
      --margin-gutter-side: auto;
      float: none !important;
      max-width: rem(340);
    }
```

**Review by urchykli — CHANGES_REQUESTED** [review]
This is looking really good, Bob! 

**Note**: I added the correct full and wide width mixins and the background color to make it easier to see what you're doing, so you'll need to do a git pull.  

Other than the few inline comments, I noticed the following things.

- [x] Caption styling for images and videos
- [x] Yellow line above the caption for images.
- [x] The right and left aligned elements should match up with the edge of wide aligned elements, so we'll need to adjust those a little.  
- [ ] Blockquote blocks are adding `<meta charset="utf-8">` to the beginning of the quote on my local and on dev.

Thank you!

**Pull request merged**